README.md

```diff
@@ -0,0 +1,148 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[contact\_form\] Contact Form](#-contact_form-contact-form)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [contact_form] <a name="about-project">Contact Form</a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[contact_form]** is a basic project to learn how the code review process will go.
```

Comment:
This section of the template is meant to guide you in making your custom input based on the project. Kindly omit it and replace it with the description on the next line which state `  is a basic project to learn how the code review process will go..` 😃 . And do make it a little bit more descriptive so those viewing your project will have a general overview. 👍